src/schemas/ProgramDonorSummary/gqlTypeDefs.ts

```diff
@@ -0,0 +1,97 @@
+import { gql } from 'apollo-server-express';
+
+export default gql`
+  scalar DateTime
+
+  enum DonorReleaseStatus {
+    FULLY_RELEASED
+    PARTIALLY_RELEASED
+    NO_RELEASE
+    UNKNOWN
+  }
+
+  enum DonorProcessingStatus {
+    COMPLETED
+    PROCESSING
+    REGISTERED
+    UNKNOWN
+  }
```

Comment:
@jaserud can you provide the actual values we're expecting for these Enums? :) it'd be nice if it matches up with the actual system

Reply:
This looks fine to me. I think there is a bit of uncertainty about how these states will be updated. I know `REGISTERED` will come from clinical but I can't say for the rest.

Reply:
how about the values for `DonorReleaseStatus` above? :)

Reply:
Ha, I know even less about those :stuck_out_tongue:

Reply:
right right, we don't have them defined :P